business/istio_validations.go

```diff
@@ -108,14 +108,14 @@ func (in *IstioValidationsService) GetValidations(ctx context.Context, namespace
 
 func (in *IstioValidationsService) getAllObjectCheckers(namespace string, istioConfigList models.IstioConfigList, exportedResources kubernetes.ExportedResources, services models.ServiceList, workloadsPerNamespace map[string]models.WorkloadList, workloads models.WorkloadList, mtlsDetails kubernetes.MTLSDetails, rbacDetails kubernetes.RBACDetails, namespaces []models.Namespace, registryServices []*kubernetes.RegistryService) []ObjectChecker {
```

Comment:
In this method, perhaps the services are not needed.
If I read the PR correctly.

Reply:
agree